src/common/style/icon_color_css.ts

```diff
@@ -1,5 +1,23 @@
 import { css } from "lit-element";
 
+export interface StateColor {
+  state: string;
+  color?: string;
+}
+
+export const computeCustomStateColor = (
+  state: string,
+  stateColors: StateColor[]
+): string | undefined => {
+  for (const stateColor of stateColors) {
+    if (stateColor.state === state) {
+      return stateColor.color || "var(--paper-item-icon-active-color, #fdd835)";
```

Comment:
We can't insert a default here like this. This is a common utility and it doesn't know where it's used.

Reply:
I don't follow why this should be avoided? I don't have problem making color required just not clear on your thought. Thanks.

Reply:
By hardcoding the CSS var, you're assuming that wherever it will be used, this is the color that's needed. This might not be the case since this is a common utility.